Xamarin.Forms.Core/Items/ItemSizingStrategy.cs

```diff
@@ -3,6 +3,7 @@
 	public enum ItemSizingStrategy
 	{
 		MeasureAllItems,
```

Comment:
These probably should have been "MeasureAll" and "MeasureFirst" to make it a little less verbose. But ok :)

Reply:
Still under the flag so we can change it .. @hartez ?